mock/el8.cfg

```diff
@@ -1,67 +1,14 @@
-config_opts['root'] = 'pulpcore-el8-x86_64'
-config_opts['target_arch'] = 'x86_64'
-config_opts['legal_host_arches'] = ('x86_64',)
-
-config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep vim python38-rpm-macros'
-config_opts['dist'] = 'el8'  # only useful for --resultdir variable subst
-config_opts['releasever'] = '8'
-config_opts['package_manager'] = 'dnf'
-config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
-config_opts['bootstrap_image'] = 'centos:8'
-
-
-config_opts['dnf.conf'] = """
-[main]
-keepcache=1
-debuglevel=2
-reposdir=/dev/null
-logfile=/var/log/yum.log
-retries=20
-obsoletes=1
-gpgcheck=0
-assumeyes=1
-syslog_ident=mock
-syslog_device=
-metadata_expire=0
-mdpolicy=group:primary
-best=1
-protected_packages=
-module_platform_id=platform:el8
-
-[BaseOS]
-name=CentOS-$releasever - Base
-mirrorlist=http://mirrorlist.centos.org/?release=8&arch=$basearch&repo=BaseOS&infra=$infra
-failovermethod=priority
-gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
-gpgcheck=1
-skip_if_unavailable=False
+include('centos-stream-8-x86_64.cfg')
 
-[PowerTools]
-name=CentOS-$releasever - PowerTools
-mirrorlist=http://mirrorlist.centos.org/?release=8&arch=$basearch&repo=PowerTools&infra=$infra
-#baseurl=http://mirror.centos.org/centos/$releasever/PowerTools/$basearch/os/
-gpgcheck=1
-gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
-
-[extras]
-name=CentOS-$releasever - Extras
-mirrorlist=http://mirrorlist.centos.org/?release=8&arch=$basearch&repo=extras&infra=$infra
-#baseurl=http://mirror.centos.org/centos/$releasever/extras/$basearch/os/
-gpgcheck=1
-enabled=1
-gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
-
-[koji-non-modular]
-name=koji-non-modular
-baseurl=http://koji.katello.org/kojifiles/releases/split/yum/koji-modules/koji/staged/x86_64/RHEL-8-001-mod/non_modular/
-module_hotfixes=1
-
-[koji-python-38]
-name=koji-python-38
-baseurl=http://koji.katello.org/kojifiles/releases/split/yum/koji-modules/koji/staged/x86_64/RHEL-8-001-mod/python38:3.8/
-module_hotfixes=1
+config_opts['root'] = 'pulpcore-el8-x86_64'
+config_opts['chroot_setup_cmd'] += " python38-rpm-macros"
```

Comment:
```suggestion
config_opts['chroot_setup_cmd'] += " python39-rpm-macros"
```

Reply:
I copied it from 3.18. Was 3.18 still Python 3.8 or was that already 3.9?

Reply:
Doh - looks like 3.18 was 3.9